var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -133,13 +133,16 @@ def configure_args(self):
                 '--with-mpi-libs= '
             ])
 
+        config_fcflags = config_cflags.copy()
+        if spec.satisfies('%gcc@10.1.0:+fortran'):
```

Comment:
```suggestion
        if spec.satisfies('%gcc@10:+fortran'):
```
Just a minor change to be consistent with other packages.